rotkehlchen/types.py

```diff
@@ -399,6 +399,17 @@ def is_bitcoin(self) -> bool:
     def is_substrate(self) -> bool:
         return self in get_args(SUPPORTED_SUBSTRATE_CHAINS)
 
+    def get_chain_type(self) -> 'str':
```

Comment:
Why using `'str'` instead of `str`?

```suggestion
    def get_chain_type(self) -> str:
```

Reply:
ah lol no idea why